operators/advanced_network/advanced_network/types.h

```diff
@@ -391,6 +391,8 @@ struct NetworkConfig {
   std::unordered_map<std::string, MemoryRegionConfig> mrs_;
   std::vector<InterfaceConfig> ifs_;
   uint16_t debug_;
+  uint32_t tx_meta_buffers_;
+  uint32_t rx_meta_buffers_;
```

Comment:
per offline discussion 

> they'll get an error that they're running out of meta buffers if this is a high burst-per-second rate.

If we can indicate in that error something like below, and add documentations for it in our config docs, then we're golden:
```
"Running out of meta buffers due to high rates... either increase your number of metadata buffers (current: x [default]) with `<tx|rx>_meta_buffers` (will increase memory usage) or increase your `batch_size` for port X queue X (will increase latency)"
```